docs/docs/how-to/creating-packages.mdx

```diff
@@ -115,4 +115,77 @@ The Instant OpenHIE executable will look for these scripts and ensure that they
 
 ## How to execute your new package
 
-Mount your package into the openhie/instant container when it is run to start the packages. It will automatically detect packages folders in the `/instant` folder and will execute them as described above. For a package to be detected it must have an `instant.json` file in the root folder of the package.
+The following mechanism for adding a package is useful for package development.
+
+> The preferred method for adding custom packages to your Instant instance will be to add the package repo GitHub url to the config - however this **feature is still in development**.
+
+### Yarn (Dev)
+
+Copy your package into the root directory of this Instant OpenHIE project.
+
+Run as normal:
+
+```sh
+yarn instant init <your_package_name>
+```
+
+### Docker
+
+To add a custom package to your instant instance, use the following flag
+
+- `-c`
+- `--custom-package`
+
+> We hope to support package url references soon
+
+To add multiple custom packages, list each package location with the flag.
+
+`-c="/path/to/package_1" -c="/path/to/package_2"`
+
+Below is an example of an entire `init` command.
+If you had downloaded the [who-covid19-surveillance-package](https://github.com/jembi/who-covid19-surveillance-package) repository onto your machine you could reference it as follows:
+
+```sh
+yarn docker:instant init core covid19surveillance -c="../who-covid19-surveillance-package"
```

Comment:
Maybe we should mention that they could also just copy the `deploy.sh` script to their own repository if they didn't want to clone the entire instant repo. That might be a better way for 3rd party package developers not to have to worry about the instant repo at all.

Reply:
Good plan will add that now 👍🏼